### ISSUE TITLE
gitex.cmd is not included in portable zip file

### ISSUE THREAD

**Issue by jbodart-argenx** [gitextensions/gitextensions#9973]
### Environment

- GitExtensions version: v3.5.4
- GIT version: NA
- OS version: NA
- .NET version: NA

### Issue description

`gitex.cmd` is not included in portable zip file.  I don't think this is on purpose.

### Steps to reproduce

download and open the zip file [GitExtensions-Portable-3.5.4.12724-65f01f399.zip](https://github.com/gitextensions/gitextensions/releases/download/v3.5.4/GitExtensions-Portable-3.5.4.12724-65f01f399.zip) 

### Did this work in previous version of GitExtensions?

the issue already existed in https://github.com/gitextensions/gitextensions/releases/download/v3.4/GitExtensions-Portable-3.4.0.9608-a4f574411.zip

**Comment by RussKie** [comment]
How do you expect to use the script from the portable "install"?

**Comment by jbodart-argenx** [comment]
unzip to selected folder, add gitex.cmd to the path, call gitex from the command line

**Comment by RussKie** [comment]
If you're on a command line in the portable folder you can call
Gitextensions.exe, no?

**Comment by jbodart-argenx** [comment]
Yes, but the documentation describes the arguments for use of gitex.cmd / gitex on the command line, not for use of Gitextensions.exe on the command line.
After having installed the .msi on my home computer and looked at the contents, it seems the same arguments could be used directly with Gitextensions.exe (but I didn't know that before), that would be an undocumented usage, and the command would be longer to type :-)

**Comment by jbodart-argenx** [comment]
so my suggestion would be: 

- either update the documentation and describe command-line use of Gitextensions.exe (instead or in addition to giftex.cmd)
- or include gifted.cmd in the portable zip file
